pkg/cmd/cli/secrets/add_secret_to_obj.go

```diff
@@ -42,11 +47,36 @@ var (
 	MountType SecretType = "mount"
 )
 
+type SecretTypeList []SecretType
```

Comment:
don't need a custom type for this... just use util.StringList and do validation on the items in the list